### PR TITLE
Navigation Header for Mobile

### PR DESCRIPTION
**Pull request by BrigetteDoelp**
# Pull Request 
​
#### What’s this PR do? 
This formats the navigation header for the mobile view.
​
#### Where should the reviewer start?
In the styles.css file on line 151, ending at line 174. This header is not perfectly formatted according to the comp, so there are commented out lines of code that will be reviewed at a future date.

#### How should this be manually tested?
Open index.html to view the header, then change the view to mobile view with the dev tools.
​
#### Any background context you would like to provide?
We as a team tackled different sections of the CSS to style. This is the first time we are dividing and conquering, and some kinks will be worked out with our git workflow as time goes on.
​
#### What are the relevant tickets?
1. Blake will have a ticket for the idea input section.
2. Next steps will to divide and conquer for all sections in larger breakpoints.

#### Screenshots (if appropriate): 
alt
​

### CODE REVIEW

**Comment by jaypeasee** [comment]
As you noted as well, I'm seeing some discrepancies from the comp. Here's what I'm noticing:
1. Too much bottom padding below the `<h1/>`
2. Too much right margin on the `<h1/>`
3. The Hamburger Icon and the `<h1/>` are not lining up exactly horizontally. I think some default styles might be overriding the flex properties you applied, so look into that maybe.
4. The `<h1>` looks relatively small compared to other text elements on the page, but we can address that later when we standardize our fonts as a group.

While there are a few of these styling discrepancies, I think we can move forward with merging this PR because it is not conflicting or changing anything previously applied to the master. The list above will hopefully help with your next iteration.

File: styles.css [hunk 1]
```diff
@@ -144,17 +149,32 @@ img {
...
+    display: inline;
+    margin-right: 8.2em;
+    }
+
+  .hamburger-button {
+    background-color: transparent;
+    border: none;
+    margin-right: 5.5em;
+    width: 3em;
+    height: 2.5em;
```
Comment (BlakeDonn) [inline styles.css:164]: When we go to refactor this probably try and stray away from hard values for margins and padding, though it is the header so not the biggest deal.

File: styles.css [hunk 2]
```diff
@@ -144,17 +149,32 @@ img {
...
   }
 
   .menu {
     height: 40px;
-    text-align: center;
+    /* text-align: center; */
     width: 100%;
+    flex-direction: row;
+    /* justify-content: space-between; */
+    align-items: stretch;
```
Comment (BlakeDonn) [inline styles.css:173]: We can probably look into stretch here, may be a different flex property we can use.

**Comment by BlakeDonn** [comment]
High level overview: I agree with JP here on some of these margins and other stylings, I think we can find a solution possibly in flex or grid. I think this merge is acceptable for the master, and gives us a great direction to follow for getting our header set up.

**Pull request merged**